app/views/layouts/angular.html.haml

```diff
@@ -20,11 +20,72 @@
+      %section{layout: "row", ng_controller: 'SidebarController', ng_cloak: "", ng_if: 'isLoggedIn()'}
+        %md-sidenav.md-sidenav-left{md_component_id: 'left', md_is_open: 'showSidebar', md_is_locked_open: '$mdMedia("gt-md") && showSidebar', md_whiteframe: 4}
+          %md_content.sidebar__content{flex: "", role: "navigation", layout: "column", ng_click: "sidebarItemSelected()"}
+            %md_list.sidebar__list.sidebar__threads{layout: "column"}
+              %md_list_item.sidebar__list-item--recent{ng_class: "{'selected': onPage('dashboardPage')}"}
+                %md_button.sidebar__list-item-button{href: "/dashboard", ng_click: "isActive()", "aria-label" => "{{ 'sidebar.recent' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.fa.fa-lg.fa-clock-o
+                  %span{translate: "sidebar.recent"}
+              %md_list_item.sidebar__list-item--unread{ng_class: "{'selected': onPage('inboxPage')}"}
+                %md_button.sidebar__list-item-button{href: "/inbox", ng_click: "isActive()", "aria-label" => "{{ 'sidebar.unread' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.i.fa.fa-lg.fa-inbox
+                  %span{translate: "sidebar.unread"}
+                  %span ({{unreadThreadCount()}})
+              %md_list_item.sidebar__list-item--muted{ng_class: "{'selected': onPage('dashboardPage', nil, 'show_muted')}"}
+                %md_button.sidebar__list-item-button{href: "/dashboard/show_muted", ng_click: "isActive()", "aria-label" => "{{ 'sidebar.muted' | translate }}"}
+                  %md_avatar_icon.sidebar__list-item-icon.i.fa.fa-lg.fa-times-circle
+                  %span{translate: "sidebar.muted"}
+            %md_divider.sidebar__divider
+
+            %md_list.sidebar__list.sidebar__groups{ng_class: "{'noGroups': groups().length < 1}"}
+              %md_list_item.sidebar__list-item--group{ng_repeat: "group in groups() | orderBy: 'name' track by group.id", ng_class: "{'selected': onPage('groupPage', group.key)}"}
+                %md_button.sidebar__list-item-button{href: "{{groupUrl(group)}}", "aria-label" => "{{group.name}}"}
+                  %img.md-avatar.lmo-box--tiny.sidebar__list-item-group-logo{ng_src: "{{group.logoUrl()}}"}
+                  %span {{group.name}}
+                %md_list.sidebar__list.sidebar__subgroups{ng_if: "group.subgroups().length > 0"}
+                  %md_list_item.sidebar__list-item{ng_repeat: "subgroup in group.subgroups() | orderBy: 'name' track by subgroup.id"}
+                    %md_button.sidebar__list-item-button--subgroup{href: "{{groupUrl(subgroup)}}"}
+                      %span {{subgroup.name}}
+                .sidebar__list-item-padding
+            %md_divider.sidebar__divider
+            
+            %md_list.sidebar__list.sidebar__explore
+              %md_button.sidebar__list-item-button{href: "/explore", "aria-label" => "{{ 'sidebar.explore' | translate }}"}
```

Comment:
I see some of these have 'selected' ng-classes and others don't. Did you run into trouble with some of them?